docs/OVERVIEW.md

```diff
@@ -0,0 +1,78 @@
+## Introduction
+
+### What are Confidential Identities?
+
+Confidential identities in Corda provide a way to restrict the knowledge of identities within a transaction to a need-to-know basis. In a normal transaction, the participants are `Party` objects which contain the `CordaX500Name` and the identity owning `PublicKey`. Confidential identities are modelled using `AnonymousParty` objects which only contain the owning `PublicKey`. 
+
+This mechanism allows a node to carry out transactions in which the participants within it do not want to be known to the network. Only the parties involved in the transaction will be able to resolve the confidential owning key. This CorDapp provides two flows; one for generating confidential identities - `RequestKeyFlow`, and one for sharing them with another party `SyncKeyMappingFlow`.
+
+This version of confidential identities differs from the old version in that we do not store the X.509 certificates for each confidential identity. This greatly reduces the storage overhead for systems that require large quantities of confidential identities. 
```

Comment:
Maybe you can also mention that the identities are now "deniable" in that the issuer of the CI can choose not to provide a proof that they created it.